README.md

```diff
@@ -19,3 +20,22 @@ quantitatively reason with them.
 Add your answer to this markdown file. [This
 page](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions)
 might help with the notation for mathematical expressions.
+
+
+## Answer
+
+Looking at slides 34, we can see that we have quarters of a standard array that are bad for finding a good pivot, and one solid good set of numbers in the middle. That, with the right combinations, the median can be either good or bad. A pivot is bad if it gives us a number outside of the good values, it's closer to the worst case that way. I decided to label good and bad pivots like this. 
+
+[ $Lo$ | $G$ | $Hi$ ] 
+
+It's important to make a distinction between wether a bad pivot is from the high or low part of the list as in some combinations can make good median values ($Lo$ $G$ $Hi$ for example). Based on these we can find all the combinations of values that will produce a bad pivot, which is when more than one bad pivot from the same side of the list occur. And find how many times they can occur.
+
+$Lo$ $Lo$ $Lo$ | $Hi$ $Hi$ $Hi$ (each only 1 permutation exists)
+
+Because G represents half of the data, it makes it hard to compare with the other two pivots because they represent only a quarter of the data. If we split G into two sides, Low good pivots and High good pivots, it is easier to compute probablity because every choice has the same size (4 * 4 * 4) and we won't have to find weighted probabilities. 
+
+$Lo$ $Lo$ $G$ | $Hi$ $Hi$ $G$ (each 6 permutations because of low and high good pivots)
+
+$Hi$ $Hi$ $Lo$ | $Lo$ $Lo$ $Hi$ (each 3 permutations)
+
+This adds up to 20 bad cases. With 64 possible combinations We have a $31.25\% $ Chance of getting a bad median and slowing down our algorithm, and therfore a $68.75\% $ Chance of getting a good median and having a faster case. This is a little worse than twice as fast as a default pivot picking process (at random) so finding a Median of three values is actually faster. 
```

Comment:
But wait, aren't the probabilities for each of the 20 different cases different?

Reply:
They were the way I was planning it out, I was trying to explain what I was doing but it was much easier to write out a split good pivot to show the probabilities that way. Now it should make sense.